src/sonic-yang-models/yang-models/sonic-flex_counter.yang

```diff
@@ -50,9 +50,6 @@ module sonic-flex_counter {
                 leaf FLEX_COUNTER_STATUS {
                     type flex_status;
                 }
-                leaf FLEX_COUNTER_DELAY_STATUS {
```

Comment:
Can we put off the YANG model change? I am wondering if there is gap between the config change and yang model change, which will cause GCU failure in PROD